linux58-tkg/install.sh

```diff
@@ -46,13 +46,13 @@ if [ "$1" = "install" ] || [ "$1" = "config" ]; then
 
   source linux*-tkg-config/prepare
 
-  if [ $1 = "install" ] && [ "$_distro" != "Ubuntu" ]; then
-    msg2 "Variable \"_distro\" in \"customization.cfg\" hasn't been set to \"Ubuntu\""
+  if [ $1 = "install" ] && [ "$_distro" != "Ubuntu" ] && [ "$_distro" != "Debian" ]; then
```

Comment:
Here you need an `or` too: It needs to be either Ubuntu OR Debian

Reply:
My bad, I went too fast and haven't read what came after the if condition... It was good. Sorry about that.